src/components/common/button/CommonButton.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+
+interface CommonButtonProps {
+  onClick?: () => void;
+  style?: React.CSSProperties;
+  children: React.ReactNode;
+  className?: string;
+  disabled?: boolean;
+  color?: string;
+  backgroundColor?: string;
+}
+
+
+const CommonButton: React.FC<CommonButtonProps> = ({
```

Comment:
The common button component should be named PrimaryButton, SecondaryButton, etc.